commands/namespaces/add.go

```diff
@@ -25,10 +25,11 @@ var (
 // NewAddCommand returns a new command to add a new namespace.
 func NewAddCommand(l *zap.SugaredLogger) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "add",
+		Use:     "add [flags] namespace",
 		Long:    "Add a new namespace",
 		Short:   "Add a new namespace",
-		Example: `everestctl namespaces add [NAMESPACE] [FLAGS]`,
+		Example: `everestctl namespaces add --operator.mongodb=true --skip-wizard ns-1,ns-2`,
```

Comment:
The example might be confusing because it will install all 3 operators still. If the goal is to install only the psmdb then the command would be:
```
everestctl namespaces add --operator.postgresql=false --operator.xtradb-cluster=false --skip-wizard ns-1,ns-2
```

Reply:
done